README.md

```diff
@@ -0,0 +1,42 @@
+# Scheduling event sync
+
+At dxw we use a variety of different systems for tracking things like holiday
+and sickness, support rotas, and project scheduling. This project automates
+synchronising those different systems.
+
+## Manual usage
+
+Normally you should be running this on a schedule eg on Heroku, but in case you
+need to run a task manually:
+
+1. Install the dependencies via Bundler:
+
+   ```
+   $ bundle install
+   ```
+
+2. Set up your environment variables by copying `.env.example` to `.env` and
```

Comment:
I think worth adding a note here or in the `.env.example` file explaining where to get the env values from - are they in 1Password or do you need to talk to a Productive admin etc.

Reply:
I'd prefer to put the notes in the example file, but yes, good idea.